jerry-core/parser/js/js-parser.c

```diff
@@ -2254,19 +2254,22 @@ parser_set_show_instrs (int show_instrs) /**< flag indicating whether to dump by
 #endif /* PARSER_DUMP_BYTE_CODE */
 } /* parser_set_show_instrs */
 
-
 /**
  * Parse EcamScript source code
  */
 jsp_status_t
 parser_parse_script (const jerry_api_char_t *source_p, /**< source code */
                      size_t size, /**< size of the source code */
-                     ecma_compiled_code_t **bytecode_data_p) /**< result */
+                     ecma_compiled_code_t **bytecode_data_p, /**< [out] JS bytecode */
+                     jerry_api_object_t **error_obj_p __attr_unused___) /**< [out] error object */
```

Comment:
This is an output parameter. What do you suggest? If we free it here, then we are not able to print it in `main-unix`